src/couch_replicator_connection.erl

```diff
@@ -120,9 +124,8 @@ handle_cast({connection_close_interval, V}, State) ->
 % owner crashed
 handle_info({'DOWN', _Ref, process, Pid, _Reason}, State) ->
```

Comment:
Wonder how this interacts with one-monitor-per-worker. Technically we could just capture Ref and just clear the worker (if found) which matches the Ref.

If we want to be efficient and kill all workers of a particular owner (by pid), it might make sense to also demonitor + [flush] others, just for proper cleanup.